dashboard-api/internal/platform/storage/account_repository.go

```diff
@@ -0,0 +1,61 @@
+package storage
+
+import (
+	"context"
+	"database/sql"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	studiesmanager "github.com/vlab-research/vlab/dashboard-api/internal"
+)
+
+type AccountRepository struct {
+	db *sql.DB
+}
+
+func NewAccountRepository(db *sql.DB) *AccountRepository {
+	return &AccountRepository{
+		db: db,
+	}
+}
+
+// Create creates a new account object in the database
+func (r *AccountRepository) Create(
+	ctx context.Context,
+	a studiesmanager.Account,
+) (studiesmanager.Account, error) {
+
+	c, err := json.Marshal(a.ConnectedAccount.Credentials)
+	if err != nil {
+		return studiesmanager.Account{}, handleCreateError(err, a)
+	}
+
+	q := "INSERT INTO credentials (user_id, entity, key, details) VALUES ($1, $2, $3, $4)"
```

Comment:
@nandanrao just want to confirm that "Accounts" in the frontend are actually "Credentials" in the database